jobs/.env

```diff
@@ -10,3 +10,11 @@ POSTGRES_PORT=5432
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
 POSTGRES_DB=badgerdoc
+
+# S3 settings
```

Comment:
Let's avoid adding env variables into local `.env` files, better to use common `.env.example` from root. We're planning to remove all `.env`